[DBM] Update troubleshooting pg_stat_statements not in search_path (#16072)
DataDog/documentation: content/en/database_monitoring/setup_postgres/troubleshooting.md

If you specified a `dbname` other than the default `postgres` in your Agent config, you must run `CREATE EXTENSION pg_stat_statements` in that database.

> If you have created the extension in your target database and you still see this warning, the extension may have been created in a schema that is not accessible to the `datadog` user. To verify this, check which schema `pg_stat_statements` was created in:

Reviewer: ```suggestion
If you created the extension in your target database and you still see this warning, the extension may have been created in a schema that is not accessible to the `datadog` user. To verify this, run this command to check which schema `pg_stat_statements` was created in:
```

Author: thanks!

```bash
psql -h localhost -U datadog -d postgres -c "select nspname from pg_extension, pg_namespace where extname = 'pg_stat_statements' and pg_extension.extnamespace = pg_namespace.oid;"
```

> Then check which schemas are visible to the `datadog` user with:

Reviewer: ```suggestion
Then, run this command to check which schemas are visible to the `datadog` user:
```

```bash
psql -h localhost -U datadog -d <your_database> -c "show search_path;"
```

> If you do not see the `pg_stat_statements` schema in the `datadog` user's `search_path`, you will need to add it to the `datadog` user. For example:

Reviewer: ```suggestion
If you do not see the `pg_stat_statements` schema in the `datadog` user's `search_path`, you need to add it to the `datadog` user. For example:
```
